### Actividad 4

### Caso 1
**Para estimar la proporción de habitantes de una ciudad que poseen ordenador  personal se toma una muestra de tamaño n. Calcula el valor mínimo de n para  garantizar, con un nivel de confianza del 95 %, que el error de estimación no  supera el 2 %. (Como se desconoce la proporción, se hará a partir del caso más  desfavorable, que será 0,5)**

In [1]:
# n = ((Z^(2))*p*q) / (E^(2))
#Z = 95% = 1.96

Z= 1.96
p = 0.5
q = 0.5
E = 0.02

n = ((1.96**2)*0.5*0.5)/(0.02**(2))
n

2400.9999999999995

**R: Debe tomarse una muestra de 2401 para asegurar un error de estimacion no mayor a 2% con un 95% de nivel de confianza**

### Caso 2
**Un fabricante de electrodomésticos sabe que la vida media de estos sigue una  distribución  normal  con  media   100   meses  y  desviación  típica   12   meses. Determina  el  mínimo  tamaño  muestral  que  garantiza,  con  una  probabilidad de   0,98,   que  la  vida  media  de  los  electrodomésticos  en  dicha  muestra  se encuentre entre  90  y  100  meses.**

media = 100

desviacion = 12

probabilidad = 0.98 => Z_a = 2.33

intervalo = [90 < x < 100]

In [15]:
# 2.33*(12/sqrt(n)) < 10

# 2.33*12 < 10*sqrt(n)

n = ((2.33*12)/10)**2
n

7.817616000000002

### Caso 3 
**Se desea obtener la media  de una variable aleatoria que se distribuye normalmente con una desviación típica de 3,2. Para ello se toma una muestra 
de 64 individuos obteniéndose una media de 32,5.**

- a)  ¿Con  qué  nivel  de  confianza  se  puede  afirmar  que  la  media  de  la  
población está entre 31,5 y 33,5? 

In [2]:
import math

#### Tamaño de la muestra
n = 64

#### Media muestral
x_bar = 32.5

#### Desviación típica poblacional
sigma = 3.2

#### Nivel de confianza (95%)
conf_level = 0.95

#### Valor crítico de la distribución normal
z_alpha = abs(math.erfinv(conf_level)) 

#### Cálculo del error estándar de la media muestral
error_est = sigma / math.sqrt(n)

#### Cálculo del intervalo de confianza
intervalo = (x_bar - z_alpha * error_est, x_bar + z_alpha * error_est)

print(f"El intervalo de confianza al {conf_level*100:.0f}% de la media poblacional es {intervalo}")

AttributeError: module 'math' has no attribute 'erfinv'

- b)  Si  la  desviación  típica  de  la  población  fuera  3,  ¿qué  tamaño  mínimo debería tener la muestra con la cual estimamos la media poblacional si queremos que el nivel de confianza sea del 99 %, y el error admisible no supere el valor de 0,75? 